back/app/mailers/application_mailer.rb

```diff
@@ -114,7 +115,13 @@ def email_address_with_name(email, name)
   end
 
   def from_email
-    email_address_with_name(ENV.fetch('DEFAULT_FROM_EMAIL'), organization_name)
+    email = custom_from_email || ENV.fetch('DEFAULT_FROM_EMAIL')
```

Comment:
I'm not 100% sure what happens in the settings when you submit the form with Admin HQ with this value, when you leave it empty. It might not be included in the tenant settings at all, might be nil, _or_ might be the empty string.

The first 2 would work fine, but the empty string would be a problem since that wouldn't make the code fall back to DEFAULT_FROM_EMAIL. To be on the safe side, I think it's better to fall back to the DEFAULT_FROM_EMAIL in case of an empty string. It would also make it possible to revert from having a custom email, to using the default value again, since the Admin HQ UI doesn't let you make a value nil or remove it once it's been set.

So this line would become something like
```
custom_from_email.presence || ENV.fetch('DEFAULT_FROM_EMAIL')
```

Reply:
You mean this?
```
email = custom_from_email.presence || ENV.fetch('DEFAULT_FROM_EMAIL')
```